pages/user-interfaces.html

```diff
@@ -2,32 +2,21 @@
 <#include "incl/header.html">
 
 <div class="framework">
-<div class="frameworklabel">Editor-support packages</div>
+<div class="frameworklabel">Most popular user interfaces</div>
 <div class="frameworkcontent">
 
 <p>
-Coq projects can be developed with a variety of editors thanks to
+Coq projects can be developed with a variety of <strong>editors</strong> thanks to
 great support provided through user-contributed extensions.  Here, we
 list such support extensions that are actively used and maintained:
 
 <ul>
 
 <li>
-<strong>Visual Studio Code</strong> and <strong>VSCodium</strong> users can use either of
-the <a href="https://github.com/coq-community/vscoq">VsCoq
-extension</a>, which is actively maintained by several contributors as part
-of <a href="https://github.com/coq-community/manifesto">coq-community</a>, or
-the new <a href="https://github.com/ejgallego/coq-lsp">coq-lsp extension</a>,
-which is based on an implementation of the <a
-href="https://microsoft.github.io/language-server-protocol">Language
-Server Protocol</a> standard, with custom extensions.
-
-<tt>coq-lsp</tt> provides continous document checking and position-based
-document navigation (an interaction model similar to what is found in e.g.,
-Lean, and which departs from the model found in other Coq user interfaces).
-See <tt>coq-lsp's</tt> <a
-href="https://github.com/ejgallego/coq-lsp/blob/main/etc/FAQ.md">FAQ</a>
-for more information.
+  <strong>Visual Studio Code</strong> and <strong>VSCodium</strong> users can use
```

Comment:
Why is coq-lsp out of this list?

Reply:
Because this first part is about "Most popular user interfaces": it is set to change in the future when LSP-based extensions become more prevalent.

Reply:
Most popular seems like an strange notion, but if that's the criteria `coq-lsp` may have already more users than for example Coq + VIM already.

Reply:
The title may not be the most accurate. The goal was to separate the extensions that we know have been used for years already, from the ones that are still new and experimental, but probably represent the future.

It's hard to tell how many users Coqtail has, but for what this is worth (not a lot), it has twice as many stars on GitHub (compared to coq-lsp) and it had about as many users in the Coq Community Survey 2022 as jsCoq (although jsCoq has many more stars on GitHub).